README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_notification_DOM/)
+    - [DEMO LINK](https://vtroni.github.io/js_notification_DOM/)
```

Comment:
Add demo link to the PR description 
<img width="473" alt="image" src="https://github.com/user-attachments/assets/6533e78c-1011-469b-afb4-b3f098cbfb26">